docs/introduction/faq.md

```diff
@@ -36,26 +36,48 @@ a vehicle in which to make WebVR successful.
 
 ## How can I get started?
 
-[contact]: #how-can-i-get-in-touch-with-the-aframe-team
-[homepage]: https://aframe.io
+[intro]: ../introduction/index.md
+[gettingstarted]: ../introduction/getting-started.md
 [guides]: ../guides/index.md
 
-A-Frame is very extensive across different disciplines so it can be difficult
-to know how to get started. We've tried our best to compile resources to get
-your started on your journey.
+Read the [Introduction][intro] to get a deeper feel of what A-Frame is.
 
-The examples on the [homepage][homepage] act as a starting point for you to
-read, copy, and paste code to quickly get started. Each example comes with an
-accompanying guide explaining how it was built and what components were used to
-built it that will help your understanding.
+See the [Getting Started][gettingstarted] guide to get set up.
 
-Below the examples are various workflow [guides][guides] featuring different
-tools and libraries. Select a guide based on what you are comfortable with
-(e.g., whether you know how to code, whether you know a certain framework,
-whether you know how to model).
+Check out the [guides][guides] for basic tutorials.
 
-If you need additional support, [contact us][contact]! We are very interested
-in understanding learning curve difficulties.
+Past that, you can browse the resources on the [`awesome-aframe`
+repository][awesome]. If you need additional support, [ask on
+StackOverflow][stackoverflow].
+
+## How is A-Frame's performance?
+
+Historically, JavaScript and the browser have a bad reputation for performance
```

Comment:
I would take out `Javascript` and focus just on the DOM as the performance bottleneck. What about this?
`The main performance bottleneck for Web Applications comes from the DOM and its layout system which was originally designed for static pages and later re purposed for dynamic applications. A-Frame uses the DOM to represent a scene graph of a 3D application using custom elements. Those elements are not handled by the browser rendering system but managed by A-Frame and ultimately rendered with WebGL. This allows us to run 90+ FPS experiences in the HTC Vive. See [A-Painter](https://blog.mozvr.com/a-painter/) for an example of a full application`